docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: '2'
+services:
+
+  nakadi:
+    build: .
+    ports:
+     - "8080:8080"
+    depends_on:
+     - postgres
+     - zookeeper
+     - kafka
+    environment:
+      NAKADI_OAUTH2_MODE: "OFF"
+      NAKADI_FEATURETOGGLE_ENABLEALL: "true"
+      NAKADI_ZOOKEEPER_BROKERS: zookeeper:2181
+      SPRING_DATASOURCE_URL: jdbc:postgresql://postgres:5432/local_nakadi_db
+      SPRING_DATASOURCE_USERNAME: nakadi
+      SPRING_DATASOURCE_PASSWORD: nakadi
+    links:
+      - zookeeper
+      - postgres
+      - kafka
+
+  postgres:
+    image: postgres
+    ports:
+      - "5432:5432"
+    volumes:
+      - "./database/nakadi:/docker-entrypoint-initdb.d"
+    environment:
+      POSTGRES_USER: nakadi
+      POSTGRES_PASSWORD: nakadi
+      POSTGRES_DB: local_nakadi_db
+
+  zookeeper:
+    image: wurstmeister/zookeeper
+    ports:
+      - "2181:2181"
+
+  kafka:
+    image: wurstmeister/kafka
```

Comment:
Yes, it is the same for docker images
image: wurstmeister/kafka:0.9.0.1
image: wurstmeister/zookeeper:3.4.6
image: postgres:9.5